### PR TITLE
Rewrite of the section "Alternatives to ruby" thoroughly.

### PR DESCRIPTION
**Pull request by murata2makoto**
To address #23, I have completely rewritten this section.  But some references are not added yet, since I cannot find them in Specref.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Jan 23, 2022, 4:36 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2FJapan-Daisy-Consortium%2Fruby-t2s-req%2Fd6714ef7fe3fe5cfa5b032bf006524d99288fa35%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: https://rawcdn.githack.com/Japan-Daisy-Consortium/ruby-t2s-req/d6714ef7fe3fe5cfa5b032bf006524d99288fa35/index.html?isPreview=true&publishDate=2022-01-23
ReSpec version: 28.2.2
File a bug: https://github.com/w3c/respec/
Error: Error: Evaluation failed: Timeout: document.respec.ready didn't resolve in 27932ms.
    at ExecutionContext._evaluateInternal (/u/spec-generator/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (/u/spec-generator/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
    at async generateHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:221:12)
    at async toHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:92:18)
    at async Object.generate [as respec] (/u/spec-generator/generators/respec.js:14:44)
    at async /u/spec-generator/server.js:203:44

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/ruby-t2s-req%2324.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -607,31 +607,61 @@ <h3>Markup for indicating ruby annotation used as gikun or interlinear note</h3>
...
+          documents in EPUB publications, PLS dictionaries (see
+          [[?PRONUNCIATION-LEXICON]]) in EPUB publications are stored
+          externally to and referenced by XHTML content documents
+          (see <a data-cite="epub-tts-10#pls">Pronunciation Lexicons
+          section</a> in [[?epub-tts-10]]).  As of now,
+          [[spoken-html]] does not have a mechanism for associationg
+          PLS lexicons to HTML documents.
+        </p>
+        <p>
+          [[?PLS]] is a powerful mechanism for the text-to-speech of
```
Comment (himorin) [inline index.html:660]:
```suggestion
          [[?PRONUNCIATION-LEXICON]] is a powerful mechanism for the text-to-speech of
```

File: index.html [hunk 2]
```diff
@@ -607,31 +607,61 @@ <h3>Markup for indicating ruby annotation used as gikun or interlinear note</h3>
...
+	  unusual names of people and places.  In particular, every
+	  occurrence of a word or phrase is read aloud in the same way
+	  regardless of the existence of ruby.  As of now, [[?PLS]] is
```
Comment (himorin) [inline index.html:663]:
```suggestion
	  regardless of the existence of ruby.  As of now, [[?PRONUNCIATION-LEXICON]] is
```

**Comment by himorin** [comment]
For PLS, how about `PLS ([[PRONUNCIATION-LEXICON]])` or just leavel as `PLS` without spec link? (lines 645, 660, 663)
Or in line 610, add abbreviation as `[[PRONUNCIATION-LEXICON]] (PLS}`?

File: index.html [hunk 3]
```diff
@@ -607,31 +607,61 @@ <h3>Markup for indicating ruby annotation used as gikun or interlinear note</h3>
     <section>
       <h2>Alternatives to ruby</h2>
 
+      <p>[[?SSML]] and [[?PRONUNCIATION-LEXICON]] can be used for
+      providing phonemic/phonetic pronunciation of CJK ideograph
```
Comment (himorin) [inline index.html]:
```suggestion
      providing phonemic/phonetic pronunciation of CJK ideographic
```

File: index.html [hunk 4]
```diff
@@ -607,31 +607,61 @@ <h3>Markup for indicating ruby annotation used as gikun or interlinear note</h3>
...
-          However, it has been reported that attaching [[?SSML]] to CJK ideographic characters thoroughly significantly 
-          increases the authoring cost. 
-          Although [[?SSML]] has been used by some textbook publishers in Japan, it is unlikely to be widely used for trade books.      
+	  [[?SSML]] is widely used for digital textbooks by more than
+          one textbook publisher in Japan.  Meanwhile, it has been
+          reported that attaching [[?SSML]] attributes to CJK
+          ideographic characters significantly increases the authoring
+          cost.  DAISY textbooks in Japan do not use SSML, since they
+          include recorded voice.  Trade books in Japan do not use
+          SSML either.
```
Comment (himorin) [inline index.html:672]:
```suggestion
          SSML neither.
```

File: index.html [hunk 5]
```diff
@@ -607,31 +607,61 @@ <h3>Markup for indicating ruby annotation used as gikun or interlinear note</h3>
...
-          [[?SSML]] can control text-to-speech much better than ruby.
+	  [[?SSML]] uses symbol collections (such as IPA and JEITA
+	  IT-4006) to represent the sounds of human languages.
+	  Phonemic/phonetic pronunciation is represented by sequences
+	  of such symbols.
+        </p>  
+        <p>
+	  EPUB 3 allows SSML attributes to be used within XHTML content
+	  documents in EPUB publications.  In the upcoming version,
+	  EPUB 3.3, these attributes are moved to [[epub-tts-10]].
```
Comment (himorin) [inline index.html]: how about to have a spec link for EPUB 3.3?

**Comment by himorin** [comment]
Reading through the whole section, I somehow felt that words `ruby` are better to be replaced with `ruby annotation`. How do you think?

(this is the last comment)

File: index.html (hunk 4 above)
Comment (murata2makoto) [reply index.html:672]: I think that "either" is correct here.  Grammarly agrees with me.

Comment (himorin) [reply index.html:672]: Ahhh, yes. sorry for making noise here.

**Comment by murata2makoto** [comment]
@himorin 

>Reading through the whole section, I somehow felt that words ruby are better to be replaced with ruby annotation. How do you think?

I have used "ruby" and "ruby annotation" interchangeably.  Do other ruby specs use these term differently?

**Comment by himorin** [comment]
I suppose simple-ruby uses `ruby` as a concept of `ルビ`, and `ruby annotation` as characters attached to its base text. (after all proposed PR merged...)

**Comment by murata2makoto** [comment]
So, does "ruby" mean attaching ruby annotations to base text or the combination of base text and a ruby annotation?  In other words, ruby annotations never contain base text, but "ruby" refers to both base text and ruby annotations.  This makes sense.

**Comment by himorin** [comment]
Yes for `ruby annotations` points only attached characters to its base text and never contain its base text. (there seems no clear text of this definition in simple-ruby, and might better to add some...)

**Comment by r12a** [comment]
> I have used "ruby" and "ruby annotation" interchangeably. Do other ruby specs use these term differently?

I advise to keep those terms clearly separated.  I understand 'ruby' to be a general term to describe some annotated text and its annotations, equivalent to the `<ruby>` element.  For the actual annotations, specifically, i would use 'ruby annotation' (better than the ambiguous 'ruby text'), which corresponds in markup to the rt element(s).  That approach also ties in with the definitions in the CSS Ruby spec, although CSS has more definitions because of its focus on the HTML and CSS structural elements.

**Pull request closed without merging**